packages/wallets/src/features/cfd/screens/ChangePassword/ChangePassword.tsx

```diff
@@ -3,24 +3,36 @@ import { ModalStepWrapper, Tab, Tabs } from '../../../../components/Base';
 import { useModal } from '../../../../components/ModalProvider';
 import { PlatformDetails } from '../../constants';
 import MT5ChangeInvestorPasswordScreens from './InvestorPassword/MT5ChangeInvestorPasswordScreens';
-import MT5ChangePasswordScreens from './MT5ChangePasswordScreens';
+import TradingPlatformChangePasswordScreens from './TradingPlatformChangePasswordScreens';
 import './ChangePassword.scss';
 
 const ChangePassword = () => {
     const { getModalState } = useModal();
     const platform = getModalState('platform') ?? PlatformDetails.mt5.platform;
     const platformTitle = PlatformDetails[platform].title;
+
+    const isDerivX = platform === PlatformDetails.dxtrade.platform;
+
     return (
         <ModalStepWrapper title={`Manage ${platformTitle} password`}>
             <div className='wallets-change-password__modal-wrapper'>
-                <Tabs wrapperClassName='wallets-change-password__container'>
-                    <Tab title={`${platformTitle} Password`}>
-                        <MT5ChangePasswordScreens platform={platform} platformTitle={platformTitle} />
-                    </Tab>
-                    <Tab title='Investor Password'>
-                        <MT5ChangeInvestorPasswordScreens />
-                    </Tab>
-                </Tabs>
+                <div className='wallets-change-password__container'>
+                    {isDerivX ? (
+                        <TradingPlatformChangePasswordScreens platform={platform} platformTitle={platformTitle} />
```

Comment:
I believe it will be the same. I wont passed as many props but I will have to define the title in the component. Whereas the current one, defines the title in the HOC and passed it as props. So I feel it's a matter of preference 🙂